cocotb_build_libs.py

```diff
@@ -36,6 +127,20 @@ def run(self):
         def_dir = os.path.join(cocotb_share_dir, "def")
         self._gen_import_libs(def_dir)
 
+        if os.name == "nt":
+            create_rc_file("libcocotbutils", "libcocotbutils.dll", ["libgpilog"])
+            create_rc_file("libgpilog", "libgpilog.dll", [])
+            create_rc_file("libcocotb", "libcocotb.dll", ["libcocotbutils", "libgpilog"])
+            create_rc_file("libgpi", "libgpi.dll", ["libcocotbutils", "libgpilog", "libcocotb"])
+            create_rc_file("simulator", _build_ext.get_ext_filename(self, "simulator"), ["libcocotbutils", "libgpilog", "libgpi"])
+            create_rc_file("libcocotbvpi_icarus", "libcocotbvpi_icarus.vpl", ["libgpi", "libgpilog"])
+            create_rc_file("libcocotbvpi_modelsim", "libcocotbvpi_modelsim.dll", ["libgpi", "libgpilog"])
+            create_rc_file("libcocotbvpi_ghdl", "libcocotbvpi_ghdl.dll", ["libgpi", "libgpilog"])
+            create_rc_file("libcocotbvpi_aldec", "libcocotbvpi_aldec.dll", ["libgpi", "libgpilog"])
+            create_rc_file("libcocotbvhpi_aldec", "libcocotbvhpi_aldec.dll", ["libgpi", "libgpilog"])
+            create_rc_file("libcocotbfli_modelsim", "libcocotbfli_modelsim.dll", ["libgpi", "libgpilog"])
+            create_sxs_appconfig(self.get_ext_fullpath(os.path.join("cocotb", "simulator")))
```

Comment:
Can you construct this by walking over the list of extension modules, rather than repeating the dependencies here? I'm ok if we still have to special case `simulator` somehow.